test/model/evaluation.jl

```diff
@@ -1,4 +1,37 @@
-@testset "controlling sampling behavior for conditioned variables" begin
+function _logjoint(model::JuliaBUGS.BUGSModel)
+    _, log_densities = JuliaBUGS.evaluate_with_env!!(model; transformed=model.transformed)
+    return log_densities.tempered_logjoint
+end
+
+function verify_log_densities_structure(log_densities)
+    @test isa(log_densities, NamedTuple)
```

Comment:
```suggestion
    @test isa(log_densities, NamedTuple)
    @test length(log_densities) == 3
```
Getting quite nitpicky here, but if we are checking the keys why not check that there are no other keys.